<launch>

  <node pkg="iter_app" type="timing.py" name="timing" output="screen">
    <param name="fake_time" value="true" />
  </node>

  <include file="$(find rad_ui)/launch/main.launch"/>

</launch>
